static/css/devhub/base.less

```diff
@@ -0,0 +1,112 @@
+@import '../common/normalize.css';
+@import '../impala/fonts/fira/fira.less';
+
+
+// Colors
+@primary-font: 'Fira Sans', sans-serif;
+@primary-font-color: #fff;
+
+@button-border: #979797;
+@primary-button-background: #29C978;
+@primary-section-background: #0568BA;
+
+
+// Common styles
+section {
+  padding: 0 20px 60px 20px;
+
+  & > h1 {
+    font-size: 40px;
+    line-height: 1.2;
+    margin-bottom: 0;
+    margin-top: 20px;
+
+    &:first-child {
+      margin-top: 0px;
+    }
+  }
+
+  & > p {
+    font-weight: 300;
+    margin-top: 10px;
+  }
+}
+
+.Button {
+  border-radius: 2px;
+  border: 1px @button-border;
+  box-shadow: 0 2px 3px 0 rgba(0, 0, 0, 0.3);
+  color: @primary-font-color;
+  display: inline-block;
+  padding: 10px 30px;
+  text-align: center;
+  text-decoration: none;
+
+  &.Button--primary {
+    background-color: @primary-button-background;
+  }
+}
+
+// Sizes
+
+// Fonts
+* {
+  font-family: @primary-font;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.DevHub-Header {
+  background-color: @primary-section-background;
+
+  h1, p {
```

Comment:
Having one selector per line is always handy. Small element selectors like this aren't too bad but it's a good habit one you have longer classes. Scanning horizontally is much harder than scanning vertically once the line size is long enough.

We use stylelint to enforce this on addons-frontend, which might be something to think about later.